au3/libraries/lib-wave-track/WaveClip.cpp

```diff
@@ -671,6 +684,19 @@ bool WaveClip::HasPitchOrSpeed() const
    return !StretchRatioEquals(1.0) || GetCentShift() != 0;
 }
 
+bool WaveClip::GetStretchEnabled() const
+{
+   return mStretchEnabled;
+}
+
+void WaveClip::SetStretchEnabled(bool enabled)
+{
+   if (mStretchEnabled == enabled) {
+      return;
+   }
+   mStretchEnabled = enabled;
```

Comment:
I'm a bit concerned about that. Consider we have stretch enabled, then set non-1 stretch ratio, and then disable stretch. 
From the api standpoint the clip will have some stretching applied, but have stretch disabled. This will be a bit ambiguous.
Also there is another function called `HasPitchOrSpeed`

Reply:
Going through the PR one more time, it seems to make sense to call the new member `mStretchSyncEnabled`, `mProjectStretchSyncEnabled` or something like that. What do you think?

Reply:
Sounds better to me too 👍

Reply:
Yeah, probably. On another topic, we reset the current clip tempo to the current project tempo once this guy is set to true

Reply:
Fixed resetting to current project tempo + added saving clipTempo as well